util/find-doc-nits

```diff
@@ -178,15 +178,15 @@ sub check()
         close POD;
     }
 
-    # Check if EXAMPLES is located after RETURN VALUES section.
-    &check_section_location($filename, $contents, "RETURN VALUES", "EXAMPLES") if $filename =~ m|man3/|;
-    # Check if HISTORY is located after SEE ALSO
-    &check_section_location($filename, $contents, "SEE ALSO", "HISTORY") if $filename =~ m|man3/|;
-    # Check if SEE ALSO is located after EXAMPLES
-    &check_section_location($filename, $contents, "EXAMPLES", "SEE ALSO") if $filename =~ m|man3/|;
-
     my $id = "${filename}:1:";
 
+    # Check ordering of some sections in man3
+    if ( $filename =~ m|man3/| ) {
+        &section_follows($id, $contents, "RETURN VALUES", "EXAMPLES");
```

Comment:
The function name is very confusing, because "follows" reverses the order:  I'm tempted to read it as

```
    section_follows('A', 'B')   =>  Section 'A'  follows section 'B'   =>   First comes section 'B', then section 'A' 
```
My alternative proposal would be to rename `check_section_location()` (which is admittedly not very expressive) to `check_section_order()` .

Reply:
Name changed to ```check_section_location``` and fixup commit pushed.
(I avoided the temptation to name it ```section_preceeds``` :)